webapp/lib/view.dart

```diff
@@ -211,26 +212,42 @@ class ConfigurationViewTagListPartial {
       ..classes.add('configure-package__tags-list');
   }
 
-  void renderTagList(List<String> tags) {
+  void renderTagList(Map<String, bool> tags) {
     tagListElement.children.clear();
-    tags.forEach((tag) {
+    tags.forEach((tag, state) {
       var tagItem = new Element.li()
         ..classes.add('configure-package__tag-item')
         ..text = tag;
       tagItem.onClick.listen((event) {
         var selectedTag = (event.target as Element);
         controller.command(controller.UIAction.configurationTagSelected, new controller.ConfigurationTagData(selectedTag: selectedTag.text.trim()));
-        tagListElement.children.forEach((t) => t.classes.remove('configure-package__tag-item-active'));
-        selectedTag.classes.add('configure-package__tag-item-active');
+      });
+      tagItem.onDragEnter.listen((event) => event.preventDefault());
+      tagItem.onDragOver.listen((event) {
+        event.preventDefault();
+        (event.target as Element).classes.add('configure-package__tag-item-active');
+      });
+      tagItem.onDragLeave.listen((event) => (event.target as Element).classes.remove('configure-package__tag-item-active'));
```

Comment:
it might be good to separate the style between selection, ie. `tag-item-active` (which should probably be renamed to `tag-item--active` since active is a "modifier") and acting as a drop target. (the difference can be just conceptual, even though the background colour for now can be the same). So I suggest that you use a different class here, for example `tag-item--drop-target`

Reply:
Can I change this in the PR I was supposed to do the other BEM fixes?

Reply:
sure, yes, that makes sense